src/target/llvm/llvm_module.cc

```diff
@@ -239,7 +239,7 @@ class LLVMModuleNode final : public runtime::ModuleNode {
     if (mtarget != nullptr) {
       llvm::MDString* pstr = llvm::dyn_cast<llvm::MDString>(mtarget);
       CHECK(pstr != nullptr);
-      target_ = pstr->getString();
+      target_ = pstr->getString().str();
```

Comment:
std::string(ptr->getString()) should work well on all llvm versions

Reply:
They are the same:

https://github.com/llvm-mirror/llvm/blob/master/include/llvm/ADT/StringRef.h#L232
https://github.com/llvm-mirror/llvm/blob/master/include/llvm/ADT/StringRef.h#L260